tests/basic/tests.py

```diff
@@ -691,6 +691,7 @@ class ManagerTest(SimpleTestCase):
         "order_by",
         "select_for_update",
         "select_related",
+        "ordered",
```

Comment:
The ticket mentions that `ordered` doesn't make sense on a manager and looking at the implementation I agree. I also don't think `query` (below) makes sense.

Unfortunately I think fixing this would add additional complexity to this PR, and at that point I'm not sure it's worth fixing, for much the same reasons expressed in the original ticket - having a property making queries feels like a footgun to me and method syntax is available.